docs/modules/ROOT/pages/references/quality-requirements/reliability/portal-api-data-recoverability.adoc

```diff
@@ -0,0 +1,26 @@
+= Data compatibility with external components
+
+Source::
+{product} administrator
+
+Stimulus::
+Reinstall APPUiO Global from scratch
```

Comment:
I think I know where the misunderstanding could come from. "Reinstall APPUiO Global from scratch" is too broad...
```suggestion
Reinstall APPUiO Global third-party component from scratch
```